hack/test/dashboards/deploy/control-plane-collector-config.yaml

```diff
@@ -303,28 +303,6 @@ data:
       # Kubernetes component auto discovery
       ##########################################################################
 
```

Comment:
Need clarification here before porting to PR #366: Why are we removing the `dns` and `coredns` metrics  below? Is that a necessity for deploying to demo environment in vmware sandbox?